src/subscription_manager/cli_command/refresh.py

```diff
@@ -41,7 +41,15 @@ def _do_command(self):
             # remove content_access cache, ensuring we get it fresh
             content_access = inj.require(inj.CONTENT_ACCESS_CACHE)
             if content_access.exists():
+                print('aaaaaa')
```

Comment:
I guess, this shouldn't be part of the final solution :-)

Reply:
Thank you for taking a look and for catching that!!